examples/multiplayer-lan/rust/src/multiplayer_controller.rs

```diff
@@ -0,0 +1,303 @@
+use core::time;
+use std::collections::HashMap;
+use std::thread;
+
+use godot::classes::{
+    Button, Control, ENetMultiplayerPeer, IControl, LineEdit, MultiplayerApi, RichTextLabel,
+};
+use godot::global::Error;
+use godot::obj::WithBaseField;
+use godot::prelude::*;
+
+use crate::scene_manager::SceneManager;
+use crate::NetworkId;
+
+const LOCALHOST: &str = "127.0.0.1";
+const PORT: i32 = 8910;
```

Comment:
stupid question, but shouldn't it be unsigned int 32b?

Reply:
Port numbers are usually a `u16`, e.g. [`std::net::SocketAddr::new`](https://doc.rust-lang.org/std/net/enum.SocketAddr.html#method.new).  In Godot it's passed as a string, though.  Maybe it'd be nice if we changed that string for a `SocketAddr` at one point.